background.js

```diff
@@ -1,16 +1,31 @@
 const oldReddit = "https://old.reddit.com";
+const excludedSubdomains = [
+  "blog.reddit.com",
+  "mod.reddit.com",
+  "oauth.reddit.com",
+  "out.reddit.com"];
```

Comment:
This doesn't do anything - the `chrome.webRequest.onBeforeRequest.addListener(` callback is only called when one of the urls listed in `urls` below is matched. Note that I have just merged https://github.com/tom-james-watson/old-reddit-redirect/pull/31 which adds `amp.reddit.com`.

So yeah - basically you don't need this `excludedSubdomains` logic.

Reply:
It looks like you've copied this from https://github.com/tom-james-watson/old-reddit-redirect/pull/5 but you haven't updated the urls to `"*://*.reddit.com/*"` to compensate.

Reply:
That change would only be necessary if we want to be able to redirect certain old.reddit.com urls to reddit.com urls, like you had originally suggested should be done for `user/overview`.